OpenNefia.Content/UI/Layer/MorePrompt.cs

```diff
@@ -18,6 +18,8 @@ public sealed class MorePrompt : UiLayerWithResult<UINone, UINone>
 
         private IAssetInstance AssetMorePrompt;
 
+        public override int? DefaultZOrder => int.MaxValue;
```

Comment:
Probably make the HUD's z-order a `const int` so you can reference it statically, then do `HudZOrder + 10000` or something.

Reply:
And in the future, maybe instead of numeric z-order we could have ordering based on widget type like with ECS event handlers.

Reply:
changed in 9eb449f